### ISSUE TITLE
训练效果异常

### ISSUE THREAD

**Issue by karmueo** [Tianxiaomo/pytorch-YOLOv4#74]
按照您提供的coins数据集和参数进行训练,训练效果不理想,loss_xy几乎不收敛,不知道是哪里出来问题.
最后一个Epoch的输出为:
loss (batch)=685, loss_cls=29.6, loss_l2=24.8, loss_obj=531, loss_wh=2.4, loss_xy=122, lr=0.001

**Comment by love2learn-7179** [comment]
I also have the same issue of loss_xy not converging. I was trying to fine-tune the model w/ coco val data. Is it because the current code uses bce for loss_xy?

![image](https://user-images.githubusercontent.com/54882021/83075117-2cdc6a00-a028-11ea-8d2c-e1f2209204df.png)
